build/azure-pipelines/build-common.yml

```diff
@@ -1,67 +1,66 @@
-parameters:

```

Comment:
I've just discovered what this diff issue is.  I turned off all auto code cleanup features in my IDE (goland), and there has been no issues.  Then I just ran gofmt -w . and pushed a commit and that has caused the problem.

So I think we need to get through this at some point, as all files should be gofmt'ed.  I can undo this local commit if needed, and then create a seperate PR that gofmt's the code, we should then add a pipeline break if gofmt . fails.

Reply:
the Go extension for vscode always runs fmt on save

Reply:
I have everything turned off on my IDE.  The only thing I did for this commit was run `gofmt -w .`, I specifcally had everything turned off in my IDE, and ran gofmt for this one commit to track down this issue.  I'll investigate further.

Reply:
The whitespace issue is main is using CRLF and the files I am editing git is converting to LF.

I presume we want to be a LF based repo.  Agree?

If so I can make a single PR that LFs the whole repo.

Reply:
I just have this in my git client config

```
core.autocrlf=true
```
Why would we not want to use that?

Reply:
Besides the test files for sqlcmd need to render with `\r\n` when testing on Windows and `\n` when testing on Linux.

Reply:
The issue with requiring folks editing their git client is ensuring everyone does it (and this doesn't seem to be the default for many installs).

Reply:
We've fixed this with a .gitattributes file